frontend/admin/src/js/components/pool/PoolTable.tsx

```diff
@@ -50,30 +46,14 @@ function poolCandidatesLinkAccessor(
   );
 }
 
-function viewLinkAccessor(
-  editUrlRoot: string,
-  id: string,
-  title: Maybe<string>,
-  classifications: AccessorClassifications,
-  stream: PoolStream | null,
-  intl: IntlShape,
-) {
-  let classificationString = ""; // type wrangling the complex type into a string
-  if (classifications && classifications[0]) {
-    const grouping = classifications[0];
-    classificationString = formatClassificationString({
-      group: grouping.group,
-      level: grouping.level,
-    });
-  }
-
+function viewLinkAccessor(editUrlRoot: string, pool: Pool, intl: IntlShape) {
   return (
-    <Link href={`${editUrlRoot}/${id}`} type="link">
-      {formattedPoolPosterTitle({
-        title,
-        classification: classificationString,
-        stream,
-        intl,
+    <Link href={`${editUrlRoot}/${pool.id}`} type="link">
+      {getFullPoolAdvertisementTitle(intl, {
+        id: pool.id,
+        name: pool.name,
+        classifications: pool.classifications,
+        stream: pool.stream,
       })}
```

Comment:
We can pass the pool in directly instead of creating a new object, since its already an object with all the (properly typed!) fields:
```suggestion
      {getFullPoolAdvertisementTitle(intl, pool)}
```